.github/workflows/deploy.yml

```diff
@@ -36,21 +36,10 @@ jobs:
             echo "HEROKU_APP_NAME=${{ secrets.HEROKU_APP_NAME_STAGING }}" >> $GITHUB_ENV
           fi
 
-      - name: Set DATABASE_URL
-        run: |
-          if [[ $BRANCH_TAG = "latest" ]]
-          then
-            echo "DATABASE_URL=${{ secrets.DATABASE_URL_PRODUCTION }}" >> $GITHUB_ENV
-          else
-            echo "DATABASE_URL=${{ secrets.DATABASE_URL_STAGING }}" >> $GITHUB_ENV
-          fi
-
       - name: Deploy
         uses: akhileshns/heroku-deploy@v3.8.8
         with:
           heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
           heroku_app_name: $HEROKU_APP_NAME
           heroku_email: ${{ secrets.HEROKU_ACCOUNT_EMAIL }}
-          usedocker: true
-          docker_build_args: |
-            DATABASE_URL
+          usedocker: true
```

Comment:
@markgravity What IDE are you using? It's better to setup to add a new line at the end of the file automatically.

Reply:
Goland :D